examples/mpi_plot_simulate_evoked.py

```diff
@@ -0,0 +1,41 @@
+"""
+===============
+Simulate dipole
+===============
+
+This example demonstrates how to run simulations with MPI using HNN-core.
+"""
+
+# Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#          Sam Neymotin <samnemo@gmail.com>
+
+import os.path as op
+
+###############################################################################
+# Let us import hnn_core
+
+import hnn_core
+from hnn_core import simulate_dipole, read_params, Network, MPI_backend
+
+hnn_core_root = op.join(op.dirname(hnn_core.__file__), '..')
+
+###############################################################################
+# Then we read the parameters file
+params_fname = op.join(hnn_core_root, 'param', 'default.json')
+params = read_params(params_fname)
+
+###############################################################################
+# Create the Network instance to use for simulations
+net = Network(params)
+
+###############################################################################
+# Now let's simulate the dipole using the MPI backend.
```

Comment:
Is this making it parallel across trials or neurons? Maybe better to document somewhere

Reply:
Updated.

https://github.com/jonescompneurolab/hnn-core/blob/1cc993d6289ceb30b317a86813f4190a8205fddf/examples/plot_simulate_evoked_mpi.py#L32-L35